lldb/source/Plugins/Process/Linux/NativeRegisterContextLinuxArm64Shared.cpp

```diff
@@ -0,0 +1,63 @@
+#include "NativeRegisterContextLinuxArm64Shared.h"
+
+using namespace lldb_private::process_linux::arm64;
+
+namespace lldb_private {
+namespace process_linux {
+namespace arm64 {
+
+namespace {
+Status ReadHardwareDebugInfoHelper(int regset, ::pid_t tid,
```

Comment:
llvm style is to use a static function rather than anonymous namespaces. https://llvm.org/docs/CodingStandards.html#restrict-visibility

Though it doesn't actually show an example of doing it, I will fix that. It does say right at the end to use static.